Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt

```diff
@@ -55,6 +76,11 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
         factoryProducer = { viewModelFactory }
     )
 
+    private val homeFragmentVm: HomeFragmentViewModel by cwaViewModels(
```

Comment:
Don't add the homefragment viewmodel here, MainActivity should not directly access that.

Reply:
I wanted to use the shared ViewModel for Activity > Fragment communication, the reason is explained in the following conversation

Reply:
updated